.github/ISSUE_TEMPLATE/1-bug.md

```diff
@@ -1,7 +1,7 @@
 ---
 name: "🐞 Bug Report"
 about: Report a bug
-labels: "type: bug"
```

Comment:
We have tooling that checks for issues that begin with `scope:` when determining untriaged issues, for the label could we change it to just `needs-triage`, without the scope?

Its more of a `status`, since its not a category for the issue to fall under.

Reply:
Changed the name to just `needs-triage`